pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -770,7 +770,53 @@ cdef class _Timedelta(timedelta):
 
     @property
     def resolution(self):
-        """ return a string representing the lowest resolution that we have """
+        """
+        Return a string representing the lowest (i.e. smallest) time
+        resolution.
+
+        Each timedelta has a defined resolution that represents the lowest OR
+        most granular level of precision. Each level of resolution is
+        represented by a short string as defined below:
+
+        ============ ============
```

Comment:
I don't think this will render in a doc-string, not really sure what the standard is for a table like list

Reply:
Do you have a suggestion?
Do  bulleted lists work?
```
          - Days:         'D'	
          - Hours:        'H'	
          - Minutes:      'T'	
          - Seconds:      'S'	
          - Milliseconds: 'L'	
          - Microseconds: 'U'	
          - Nanoseconds:  'N'
```

Reply:
yep that's prob ok